### PR TITLE
Update object-model.md

### PR DESCRIPTION
**Pull request by Surrogate-TM**
Fix enumerations hyperlink

### CODE REVIEW

File: api/overview/Visio/object-model.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Use the table of contents in the left navigation to view the topics in this sect
 
 ## See also
 
-- [Visio enumerations](../../../api/visio(enumerations).md)
+- [Visio enumerations](https://docs.microsoft.com/en-us/office/vba/api/visio(enumerations))
```
Comment (lindalu-MSFT) [inline api/overview/Visio/object-model.md:22]:
```suggestion
- [Visio enumerations](/office/vba/api/visio(enumerations.md))
```

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]

File: api/overview/Visio/object-model.md (hunk 1 above)
Comment (Surrogate-TM) [reply api/overview/Visio/object-model.md:22]: @lindalu-MSFT,
I try your suggests but hyperlinks dont work at my side…
https://github.com/MicrosoftDocs/VBA-Docs/blob/main/office/vba/api/visio(enumerations.md)

**Comment by lindalu-MSFT** [comment]
Merged, will test on prod tomorrow! Thanks

**Comment by lindalu-MSFT** [comment]
Everything looks good on the live side.

**Comment by Surrogate-TM** [comment]
Yes! Thank you, Linda!

**Pull request merged**